src/lib/types/stores/strategy-store.ts

```diff
@@ -0,0 +1,36 @@
+import { Store } from './store';
+
+export interface IStrategy {
+    name: string;
+    editable: boolean;
+    description: string;
+    parameters: object;
+    deprecated: boolean;
+    displayName: string;
+}
+
+export interface IEditableStrategy {
+    name: string;
+    description: string;
+    parameters: object;
+    deprecated: boolean;
+}
+
+export interface IMinimalStrategy {
+    name: string;
+    description: string;
+    parameters: string;
+}
+
+export interface IStrategyStore extends Store<IStrategy, string> {
+    getEditableStrategies(): Promise<IEditableStrategy[]>;
+    getStrategy(name: string): Promise<IStrategy>;
```

Comment:
how is this different from the `get()` method from "Store"?

Reply:
It isn't, I was just lazy and didn't want to have to go through all parts where actually used it. I'll fix